src/native/public/mono/jit/details/mono-private-unstable-types.h

```diff
@@ -45,10 +45,15 @@ typedef struct {
 	char **dirs;
 } MonoCoreLookupPaths;
 
+typedef struct {
+	uint64_t current_local_time;
+} MonoCoreLocalTime;
```

Comment:
if we decide to use the preparsed structure,

1. it's not the _time_, it's the _UTC offset_, right?
2. we should add some comments to the header.

Reply:
Ah right, the naming was prior to when I realized we needed to grab the offset itself. I'll fix it if we do end up leveraging the preparsed structure.

Reply:
Removed the preparsed structure